[2025-01-16T18:29:45.783+0000] {processor.py:186} INFO - Started process (PID=5599) to work on /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:29:45.784+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/download_rocket_launch.py for tasks to queue
[2025-01-16T18:29:45.787+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:29:45.786+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:29:45.795+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:29:45.791+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/download_rocket_launch.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/download_rocket_launch.py", line 14, in <module>
    start_date=airflow.dates.days_ago(14),
               ^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/__init__.py", line 107, in __getattr__
    raise AttributeError(f"module {__name__!r} has no attribute {name!r}")
AttributeError: module 'airflow' has no attribute 'dates'
[2025-01-16T18:29:45.797+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:29:45.828+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/download_rocket_launch.py took 0.053 seconds
[2025-01-16T18:30:16.042+0000] {processor.py:186} INFO - Started process (PID=5663) to work on /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:30:16.043+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/download_rocket_launch.py for tasks to queue
[2025-01-16T18:30:16.046+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:30:16.045+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:30:16.050+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:30:16.049+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/download_rocket_launch.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/download_rocket_launch.py", line 14, in <module>
    start_date=airflow.dates.days_ago(14),
               ^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/__init__.py", line 107, in __getattr__
    raise AttributeError(f"module {__name__!r} has no attribute {name!r}")
AttributeError: module 'airflow' has no attribute 'dates'
[2025-01-16T18:30:16.052+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:30:16.077+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/download_rocket_launch.py took 0.041 seconds
[2025-01-16T18:30:46.453+0000] {processor.py:186} INFO - Started process (PID=5727) to work on /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:30:46.455+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/download_rocket_launch.py for tasks to queue
[2025-01-16T18:30:46.457+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:30:46.457+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:30:46.462+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:30:46.460+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/download_rocket_launch.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/download_rocket_launch.py", line 14, in <module>
    start_date=airflow.dates.days_ago(14),
               ^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/__init__.py", line 107, in __getattr__
    raise AttributeError(f"module {__name__!r} has no attribute {name!r}")
AttributeError: module 'airflow' has no attribute 'dates'
[2025-01-16T18:30:46.464+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:30:46.490+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/download_rocket_launch.py took 0.044 seconds
[2025-01-16T18:31:16.936+0000] {processor.py:186} INFO - Started process (PID=5798) to work on /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:31:16.937+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/download_rocket_launch.py for tasks to queue
[2025-01-16T18:31:16.940+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:31:16.940+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:31:16.945+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:31:16.943+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/download_rocket_launch.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/download_rocket_launch.py", line 14, in <module>
    start_date=airflow.dates.days_ago(14),
               ^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/__init__.py", line 107, in __getattr__
    raise AttributeError(f"module {__name__!r} has no attribute {name!r}")
AttributeError: module 'airflow' has no attribute 'dates'
[2025-01-16T18:31:16.946+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:31:16.970+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/download_rocket_launch.py took 0.040 seconds
[2025-01-16T18:31:47.183+0000] {processor.py:186} INFO - Started process (PID=5862) to work on /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:31:47.185+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/download_rocket_launch.py for tasks to queue
[2025-01-16T18:31:47.187+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:31:47.187+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:31:47.191+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:31:47.190+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/download_rocket_launch.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/download_rocket_launch.py", line 14, in <module>
    start_date=airflow.dates.days_ago(14),
               ^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/__init__.py", line 107, in __getattr__
    raise AttributeError(f"module {__name__!r} has no attribute {name!r}")
AttributeError: module 'airflow' has no attribute 'dates'
[2025-01-16T18:31:47.192+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:31:47.218+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/download_rocket_launch.py took 0.041 seconds
[2025-01-16T18:32:18.001+0000] {processor.py:186} INFO - Started process (PID=5925) to work on /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:32:18.003+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/download_rocket_launch.py for tasks to queue
[2025-01-16T18:32:18.006+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:32:18.005+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:32:18.011+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:32:18.009+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/download_rocket_launch.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/download_rocket_launch.py", line 14, in <module>
    start_date=airflow.dates.days_ago(14),
               ^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/__init__.py", line 107, in __getattr__
    raise AttributeError(f"module {__name__!r} has no attribute {name!r}")
AttributeError: module 'airflow' has no attribute 'dates'
[2025-01-16T18:32:18.012+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:32:18.038+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/download_rocket_launch.py took 0.042 seconds
[2025-01-16T18:32:48.170+0000] {processor.py:186} INFO - Started process (PID=5989) to work on /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:32:48.171+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/download_rocket_launch.py for tasks to queue
[2025-01-16T18:32:48.174+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:32:48.173+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:32:48.179+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:32:48.176+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/download_rocket_launch.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/download_rocket_launch.py", line 14, in <module>
    start_date=airflow.dates.days_ago(14),
               ^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/__init__.py", line 107, in __getattr__
    raise AttributeError(f"module {__name__!r} has no attribute {name!r}")
AttributeError: module 'airflow' has no attribute 'dates'
[2025-01-16T18:32:48.180+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:32:48.210+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/download_rocket_launch.py took 0.049 seconds
[2025-01-16T18:33:19.149+0000] {processor.py:186} INFO - Started process (PID=6053) to work on /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:33:19.151+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/download_rocket_launch.py for tasks to queue
[2025-01-16T18:33:19.155+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:33:19.154+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:33:19.160+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:33:19.158+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/download_rocket_launch.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/download_rocket_launch.py", line 14, in <module>
    start_date=airflow.dates.days_ago(14),
               ^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/__init__.py", line 107, in __getattr__
    raise AttributeError(f"module {__name__!r} has no attribute {name!r}")
AttributeError: module 'airflow' has no attribute 'dates'
[2025-01-16T18:33:19.162+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:33:19.190+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/download_rocket_launch.py took 0.048 seconds
[2025-01-16T18:33:49.621+0000] {processor.py:186} INFO - Started process (PID=6117) to work on /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:33:49.622+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/download_rocket_launch.py for tasks to queue
[2025-01-16T18:33:49.625+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:33:49.625+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:33:49.632+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:33:49.629+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/download_rocket_launch.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/download_rocket_launch.py", line 14, in <module>
    start_date=airflow.dates.days_ago(14),
               ^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/__init__.py", line 107, in __getattr__
    raise AttributeError(f"module {__name__!r} has no attribute {name!r}")
AttributeError: module 'airflow' has no attribute 'dates'
[2025-01-16T18:33:49.634+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:33:49.664+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/download_rocket_launch.py took 0.054 seconds
[2025-01-16T18:34:20.238+0000] {processor.py:186} INFO - Started process (PID=6180) to work on /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:34:20.240+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/download_rocket_launch.py for tasks to queue
[2025-01-16T18:34:20.243+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:34:20.243+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:34:20.248+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:34:20.246+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/download_rocket_launch.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/download_rocket_launch.py", line 14, in <module>
    start_date=airflow.dates.days_ago(14),
               ^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/__init__.py", line 107, in __getattr__
    raise AttributeError(f"module {__name__!r} has no attribute {name!r}")
AttributeError: module 'airflow' has no attribute 'dates'
[2025-01-16T18:34:20.250+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:34:20.282+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/download_rocket_launch.py took 0.050 seconds
[2025-01-16T18:34:51.121+0000] {processor.py:186} INFO - Started process (PID=6243) to work on /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:34:51.124+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/download_rocket_launch.py for tasks to queue
[2025-01-16T18:34:51.135+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:34:51.133+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:34:51.152+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:34:51.144+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/download_rocket_launch.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/download_rocket_launch.py", line 14, in <module>
    start_date=airflow.dates.days_ago(14),
               ^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/__init__.py", line 107, in __getattr__
    raise AttributeError(f"module {__name__!r} has no attribute {name!r}")
AttributeError: module 'airflow' has no attribute 'dates'
[2025-01-16T18:34:51.156+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:34:51.229+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/download_rocket_launch.py took 0.127 seconds
[2025-01-16T18:35:21.439+0000] {processor.py:186} INFO - Started process (PID=6308) to work on /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:35:21.439+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/download_rocket_launch.py for tasks to queue
[2025-01-16T18:35:21.443+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:35:21.443+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:35:21.447+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:35:21.446+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/download_rocket_launch.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/download_rocket_launch.py", line 14, in <module>
    start_date=airflow.dates.days_ago(14),
               ^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/__init__.py", line 107, in __getattr__
    raise AttributeError(f"module {__name__!r} has no attribute {name!r}")
AttributeError: module 'airflow' has no attribute 'dates'
[2025-01-16T18:35:21.448+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:35:21.479+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/download_rocket_launch.py took 0.046 seconds
[2025-01-16T18:35:51.587+0000] {processor.py:186} INFO - Started process (PID=6372) to work on /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:35:51.589+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/download_rocket_launch.py for tasks to queue
[2025-01-16T18:35:51.592+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:35:51.591+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:35:51.596+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:35:51.594+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/download_rocket_launch.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/download_rocket_launch.py", line 14, in <module>
    start_date=airflow.dates.days_ago(14),
               ^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/__init__.py", line 107, in __getattr__
    raise AttributeError(f"module {__name__!r} has no attribute {name!r}")
AttributeError: module 'airflow' has no attribute 'dates'
[2025-01-16T18:35:51.597+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:35:51.622+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/download_rocket_launch.py took 0.044 seconds
[2025-01-16T18:36:22.378+0000] {processor.py:186} INFO - Started process (PID=6437) to work on /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:36:22.380+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/download_rocket_launch.py for tasks to queue
[2025-01-16T18:36:22.383+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:36:22.383+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:36:22.389+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:36:22.387+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/download_rocket_launch.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/download_rocket_launch.py", line 14, in <module>
    start_date=airflow.dates.days_ago(14),
               ^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/__init__.py", line 107, in __getattr__
    raise AttributeError(f"module {__name__!r} has no attribute {name!r}")
AttributeError: module 'airflow' has no attribute 'dates'
[2025-01-16T18:36:22.392+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:36:22.430+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/download_rocket_launch.py took 0.058 seconds
[2025-01-16T18:36:52.659+0000] {processor.py:186} INFO - Started process (PID=6501) to work on /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:36:52.661+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/download_rocket_launch.py for tasks to queue
[2025-01-16T18:36:52.664+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:36:52.664+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:36:52.669+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:36:52.667+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/download_rocket_launch.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/download_rocket_launch.py", line 14, in <module>
    start_date=airflow.dates.days_ago(14),
               ^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/__init__.py", line 107, in __getattr__
    raise AttributeError(f"module {__name__!r} has no attribute {name!r}")
AttributeError: module 'airflow' has no attribute 'dates'
[2025-01-16T18:36:52.670+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:36:52.698+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/download_rocket_launch.py took 0.046 seconds
[2025-01-16T18:37:23.285+0000] {processor.py:186} INFO - Started process (PID=6565) to work on /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:37:23.287+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/download_rocket_launch.py for tasks to queue
[2025-01-16T18:37:23.292+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:37:23.291+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:37:23.296+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:37:23.294+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/download_rocket_launch.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/download_rocket_launch.py", line 14, in <module>
    start_date=airflow.dates.days_ago(14),
               ^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/__init__.py", line 107, in __getattr__
    raise AttributeError(f"module {__name__!r} has no attribute {name!r}")
AttributeError: module 'airflow' has no attribute 'dates'
[2025-01-16T18:37:23.297+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:37:23.326+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/download_rocket_launch.py took 0.048 seconds
[2025-01-16T18:37:53.914+0000] {processor.py:186} INFO - Started process (PID=6636) to work on /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:37:53.915+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/download_rocket_launch.py for tasks to queue
[2025-01-16T18:37:53.918+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:37:53.918+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:37:53.923+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:37:53.921+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/download_rocket_launch.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/download_rocket_launch.py", line 14, in <module>
    start_date=airflow.dates.days_ago(14),
               ^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/__init__.py", line 107, in __getattr__
    raise AttributeError(f"module {__name__!r} has no attribute {name!r}")
AttributeError: module 'airflow' has no attribute 'dates'
[2025-01-16T18:37:53.924+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:37:53.950+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/download_rocket_launch.py took 0.042 seconds
[2025-01-16T18:38:24.698+0000] {processor.py:186} INFO - Started process (PID=6700) to work on /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:38:24.703+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/download_rocket_launch.py for tasks to queue
[2025-01-16T18:38:24.707+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:38:24.707+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:38:24.711+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:38:24.709+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/download_rocket_launch.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/download_rocket_launch.py", line 14, in <module>
    start_date=airflow.dates.days_ago(14),
               ^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/__init__.py", line 107, in __getattr__
    raise AttributeError(f"module {__name__!r} has no attribute {name!r}")
AttributeError: module 'airflow' has no attribute 'dates'
[2025-01-16T18:38:24.712+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:38:24.739+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/download_rocket_launch.py took 0.048 seconds
[2025-01-16T18:38:54.787+0000] {processor.py:186} INFO - Started process (PID=6765) to work on /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:38:54.789+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/download_rocket_launch.py for tasks to queue
[2025-01-16T18:38:54.792+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:38:54.791+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:38:54.795+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:38:54.794+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/download_rocket_launch.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/download_rocket_launch.py", line 14, in <module>
    start_date=airflow.dates.days_ago(14),
               ^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/__init__.py", line 107, in __getattr__
    raise AttributeError(f"module {__name__!r} has no attribute {name!r}")
AttributeError: module 'airflow' has no attribute 'dates'
[2025-01-16T18:38:54.796+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:38:54.818+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/download_rocket_launch.py took 0.036 seconds
[2025-01-16T18:39:24.897+0000] {processor.py:186} INFO - Started process (PID=6828) to work on /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:39:24.901+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/download_rocket_launch.py for tasks to queue
[2025-01-16T18:39:24.908+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:39:24.906+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:39:24.916+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:39:24.913+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/download_rocket_launch.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/download_rocket_launch.py", line 14, in <module>
    start_date=airflow.dates.days_ago(14),
               ^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/__init__.py", line 107, in __getattr__
    raise AttributeError(f"module {__name__!r} has no attribute {name!r}")
AttributeError: module 'airflow' has no attribute 'dates'
[2025-01-16T18:39:24.918+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:39:24.978+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/download_rocket_launch.py took 0.093 seconds
[2025-01-16T18:39:32.818+0000] {processor.py:186} INFO - Started process (PID=6850) to work on /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:39:32.819+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/download_rocket_launch.py for tasks to queue
[2025-01-16T18:39:32.823+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:39:32.822+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:39:32.829+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:39:32.827+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/download_rocket_launch.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/download_rocket_launch.py", line 14, in <module>
    start_date=airflow.dates.date_ago(14),
               ^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/__init__.py", line 107, in __getattr__
    raise AttributeError(f"module {__name__!r} has no attribute {name!r}")
AttributeError: module 'airflow' has no attribute 'dates'
[2025-01-16T18:39:32.831+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:39:32.861+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/download_rocket_launch.py took 0.051 seconds
[2025-01-16T18:39:33.361+0000] {processor.py:186} INFO - Started process (PID=6857) to work on /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:39:33.362+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/download_rocket_launch.py for tasks to queue
[2025-01-16T18:39:33.366+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:39:33.365+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:39:33.372+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:39:33.370+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/download_rocket_launch.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/download_rocket_launch.py", line 14, in <module>
    start_date=airflow.dates.date_ago(14),
               ^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/__init__.py", line 107, in __getattr__
    raise AttributeError(f"module {__name__!r} has no attribute {name!r}")
AttributeError: module 'airflow' has no attribute 'dates'
[2025-01-16T18:39:33.373+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:39:33.409+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/download_rocket_launch.py took 0.059 seconds
[2025-01-16T18:39:47.991+0000] {processor.py:186} INFO - Started process (PID=6877) to work on /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:39:47.992+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/download_rocket_launch.py for tasks to queue
[2025-01-16T18:39:47.996+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:39:47.995+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:39:48.003+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:39:48.001+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/download_rocket_launch.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/download_rocket_launch.py", line 14, in <module>
    start_date=airflow.date.date_ago(14),
               ^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/__init__.py", line 107, in __getattr__
    raise AttributeError(f"module {__name__!r} has no attribute {name!r}")
AttributeError: module 'airflow' has no attribute 'date'
[2025-01-16T18:39:48.004+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:39:48.038+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/download_rocket_launch.py took 0.056 seconds
[2025-01-16T18:40:00.075+0000] {processor.py:186} INFO - Started process (PID=6914) to work on /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:40:00.076+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/download_rocket_launch.py for tasks to queue
[2025-01-16T18:40:00.079+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:40:00.078+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:40:00.083+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:40:00.081+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/download_rocket_launch.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/download_rocket_launch.py", line 14, in <module>
    start_date=airflow.utils.date.date_ago(14),
               ^^^^^^^^^^^^^^^^^^
AttributeError: module 'airflow.utils' has no attribute 'date'. Did you mean: 'dates'?
[2025-01-16T18:40:00.083+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:40:00.106+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/download_rocket_launch.py took 0.036 seconds
[2025-01-16T18:40:00.135+0000] {processor.py:186} INFO - Started process (PID=6915) to work on /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:40:00.135+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/download_rocket_launch.py for tasks to queue
[2025-01-16T18:40:00.137+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:40:00.137+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:40:00.140+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:40:00.139+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/download_rocket_launch.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/download_rocket_launch.py", line 14, in <module>
    start_date=airflow.utils.date.date_ago(14),
               ^^^^^^^^^^^^^^^^^^
AttributeError: module 'airflow.utils' has no attribute 'date'. Did you mean: 'dates'?
[2025-01-16T18:40:00.141+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:40:00.166+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/download_rocket_launch.py took 0.036 seconds
[2025-01-16T18:40:00.197+0000] {processor.py:186} INFO - Started process (PID=6916) to work on /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:40:00.198+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/download_rocket_launch.py for tasks to queue
[2025-01-16T18:40:00.201+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:40:00.200+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:40:00.204+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:40:00.202+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/download_rocket_launch.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/download_rocket_launch.py", line 14, in <module>
    start_date=airflow.utils.date.date_ago(14),
               ^^^^^^^^^^^^^^^^^^
AttributeError: module 'airflow.utils' has no attribute 'date'. Did you mean: 'dates'?
[2025-01-16T18:40:00.204+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:40:00.228+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/download_rocket_launch.py took 0.037 seconds
[2025-01-16T18:40:00.260+0000] {processor.py:186} INFO - Started process (PID=6917) to work on /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:40:00.261+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/download_rocket_launch.py for tasks to queue
[2025-01-16T18:40:00.263+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:40:00.263+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:40:00.266+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:40:00.265+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/download_rocket_launch.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/download_rocket_launch.py", line 14, in <module>
    start_date=airflow.utils.date.date_ago(14),
               ^^^^^^^^^^^^^^^^^^
AttributeError: module 'airflow.utils' has no attribute 'date'. Did you mean: 'dates'?
[2025-01-16T18:40:00.267+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:40:00.291+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/download_rocket_launch.py took 0.037 seconds
[2025-01-16T18:40:00.323+0000] {processor.py:186} INFO - Started process (PID=6918) to work on /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:40:00.323+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/download_rocket_launch.py for tasks to queue
[2025-01-16T18:40:00.326+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:40:00.325+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:40:00.330+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:40:00.329+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/download_rocket_launch.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/download_rocket_launch.py", line 14, in <module>
    start_date=airflow.utils.date.date_ago(14),
               ^^^^^^^^^^^^^^^^^^
AttributeError: module 'airflow.utils' has no attribute 'date'. Did you mean: 'dates'?
[2025-01-16T18:40:00.331+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:40:00.354+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/download_rocket_launch.py took 0.037 seconds
[2025-01-16T18:40:00.386+0000] {processor.py:186} INFO - Started process (PID=6919) to work on /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:40:00.387+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/download_rocket_launch.py for tasks to queue
[2025-01-16T18:40:00.389+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:40:00.388+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:40:00.394+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:40:00.392+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/download_rocket_launch.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/download_rocket_launch.py", line 14, in <module>
    start_date=airflow.utils.date.date_ago(14),
               ^^^^^^^^^^^^^^^^^^
AttributeError: module 'airflow.utils' has no attribute 'date'. Did you mean: 'dates'?
[2025-01-16T18:40:00.394+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:40:00.418+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/download_rocket_launch.py took 0.038 seconds
[2025-01-16T18:40:00.447+0000] {processor.py:186} INFO - Started process (PID=6920) to work on /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:40:00.448+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/download_rocket_launch.py for tasks to queue
[2025-01-16T18:40:00.450+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:40:00.450+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:40:00.453+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:40:00.452+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/download_rocket_launch.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/download_rocket_launch.py", line 14, in <module>
    start_date=airflow.utils.date.date_ago(14),
               ^^^^^^^^^^^^^^^^^^
AttributeError: module 'airflow.utils' has no attribute 'date'. Did you mean: 'dates'?
[2025-01-16T18:40:00.454+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:40:00.477+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/download_rocket_launch.py took 0.036 seconds
[2025-01-16T18:40:00.508+0000] {processor.py:186} INFO - Started process (PID=6921) to work on /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:40:00.509+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/download_rocket_launch.py for tasks to queue
[2025-01-16T18:40:00.512+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:40:00.512+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:40:00.515+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:40:00.514+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/download_rocket_launch.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/download_rocket_launch.py", line 14, in <module>
    start_date=airflow.utils.date.date_ago(14),
               ^^^^^^^^^^^^^^^^^^
AttributeError: module 'airflow.utils' has no attribute 'date'. Did you mean: 'dates'?
[2025-01-16T18:40:00.516+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:40:00.540+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/download_rocket_launch.py took 0.037 seconds
[2025-01-16T18:40:00.574+0000] {processor.py:186} INFO - Started process (PID=6922) to work on /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:40:00.575+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/download_rocket_launch.py for tasks to queue
[2025-01-16T18:40:00.579+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:40:00.579+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:40:00.583+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:40:00.581+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/download_rocket_launch.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/download_rocket_launch.py", line 14, in <module>
    start_date=airflow.utils.date.date_ago(14),
               ^^^^^^^^^^^^^^^^^^
AttributeError: module 'airflow.utils' has no attribute 'date'. Did you mean: 'dates'?
[2025-01-16T18:40:00.584+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:40:00.606+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/download_rocket_launch.py took 0.037 seconds
[2025-01-16T18:40:00.637+0000] {processor.py:186} INFO - Started process (PID=6923) to work on /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:40:00.638+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/download_rocket_launch.py for tasks to queue
[2025-01-16T18:40:00.640+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:40:00.640+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:40:00.644+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:40:00.642+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/download_rocket_launch.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/download_rocket_launch.py", line 14, in <module>
    start_date=airflow.utils.date.date_ago(14),
               ^^^^^^^^^^^^^^^^^^
AttributeError: module 'airflow.utils' has no attribute 'date'. Did you mean: 'dates'?
[2025-01-16T18:40:00.645+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:40:00.666+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/download_rocket_launch.py took 0.034 seconds
[2025-01-16T18:40:00.697+0000] {processor.py:186} INFO - Started process (PID=6924) to work on /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:40:00.697+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/download_rocket_launch.py for tasks to queue
[2025-01-16T18:40:00.700+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:40:00.700+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:40:00.703+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:40:00.702+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/download_rocket_launch.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/download_rocket_launch.py", line 14, in <module>
    start_date=airflow.utils.date.date_ago(14),
               ^^^^^^^^^^^^^^^^^^
AttributeError: module 'airflow.utils' has no attribute 'date'. Did you mean: 'dates'?
[2025-01-16T18:40:00.704+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:40:00.725+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/download_rocket_launch.py took 0.033 seconds
[2025-01-16T18:40:00.754+0000] {processor.py:186} INFO - Started process (PID=6925) to work on /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:40:00.755+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/download_rocket_launch.py for tasks to queue
[2025-01-16T18:40:00.758+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:40:00.758+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:40:00.761+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:40:00.760+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/download_rocket_launch.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/download_rocket_launch.py", line 14, in <module>
    start_date=airflow.utils.date.date_ago(14),
               ^^^^^^^^^^^^^^^^^^
AttributeError: module 'airflow.utils' has no attribute 'date'. Did you mean: 'dates'?
[2025-01-16T18:40:00.762+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:40:00.781+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/download_rocket_launch.py took 0.032 seconds
[2025-01-16T18:40:00.816+0000] {processor.py:186} INFO - Started process (PID=6926) to work on /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:40:00.816+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/download_rocket_launch.py for tasks to queue
[2025-01-16T18:40:00.819+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:40:00.819+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:40:00.824+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:40:00.822+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/download_rocket_launch.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/download_rocket_launch.py", line 14, in <module>
    start_date=airflow.utils.date.date_ago(14),
               ^^^^^^^^^^^^^^^^^^
AttributeError: module 'airflow.utils' has no attribute 'date'. Did you mean: 'dates'?
[2025-01-16T18:40:00.825+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:40:00.848+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/download_rocket_launch.py took 0.038 seconds
[2025-01-16T18:40:00.876+0000] {processor.py:186} INFO - Started process (PID=6927) to work on /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:40:00.877+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/download_rocket_launch.py for tasks to queue
[2025-01-16T18:40:00.879+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:40:00.879+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:40:00.882+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:40:00.881+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/download_rocket_launch.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/download_rocket_launch.py", line 14, in <module>
    start_date=airflow.utils.date.date_ago(14),
               ^^^^^^^^^^^^^^^^^^
AttributeError: module 'airflow.utils' has no attribute 'date'. Did you mean: 'dates'?
[2025-01-16T18:40:00.883+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:40:00.905+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/download_rocket_launch.py took 0.034 seconds
[2025-01-16T18:40:00.936+0000] {processor.py:186} INFO - Started process (PID=6928) to work on /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:40:00.936+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/download_rocket_launch.py for tasks to queue
[2025-01-16T18:40:00.938+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:40:00.938+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:40:00.942+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:40:00.940+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/download_rocket_launch.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/download_rocket_launch.py", line 14, in <module>
    start_date=airflow.utils.date.date_ago(14),
               ^^^^^^^^^^^^^^^^^^
AttributeError: module 'airflow.utils' has no attribute 'date'. Did you mean: 'dates'?
[2025-01-16T18:40:00.942+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:40:00.963+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/download_rocket_launch.py took 0.032 seconds
[2025-01-16T18:40:00.993+0000] {processor.py:186} INFO - Started process (PID=6929) to work on /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:40:00.994+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/download_rocket_launch.py for tasks to queue
[2025-01-16T18:40:00.998+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:40:00.997+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:40:01.001+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:40:01.000+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/download_rocket_launch.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/download_rocket_launch.py", line 14, in <module>
    start_date=airflow.utils.date.date_ago(14),
               ^^^^^^^^^^^^^^^^^^
AttributeError: module 'airflow.utils' has no attribute 'date'. Did you mean: 'dates'?
[2025-01-16T18:40:01.001+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:40:01.023+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/download_rocket_launch.py took 0.034 seconds
[2025-01-16T18:40:01.056+0000] {processor.py:186} INFO - Started process (PID=6930) to work on /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:40:01.057+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/download_rocket_launch.py for tasks to queue
[2025-01-16T18:40:01.060+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:40:01.060+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:40:01.064+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:40:01.063+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/download_rocket_launch.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/download_rocket_launch.py", line 14, in <module>
    start_date=airflow.utils.date.date_ago(14),
               ^^^^^^^^^^^^^^^^^^
AttributeError: module 'airflow.utils' has no attribute 'date'. Did you mean: 'dates'?
[2025-01-16T18:40:01.065+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:40:01.087+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/download_rocket_launch.py took 0.036 seconds
[2025-01-16T18:40:04.659+0000] {processor.py:186} INFO - Started process (PID=6939) to work on /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:40:04.660+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/download_rocket_launch.py for tasks to queue
[2025-01-16T18:40:04.663+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:40:04.663+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:40:04.671+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:40:04.669+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/download_rocket_launch.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/download_rocket_launch.py", line 14, in <module>
    start_date=airflow.utils.dates.date_ago(14),
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: module 'airflow.utils.dates' has no attribute 'date_ago'. Did you mean: 'days_ago'?
[2025-01-16T18:40:04.673+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:40:04.705+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/download_rocket_launch.py took 0.054 seconds
[2025-01-16T18:40:20.306+0000] {processor.py:186} INFO - Started process (PID=6959) to work on /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:40:20.307+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/download_rocket_launch.py for tasks to queue
[2025-01-16T18:40:20.314+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:40:20.312+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:40:20.334+0000] {processor.py:925} INFO - DAG(s) 'download_rocket_launch' retrieved from /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:40:20.477+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:40:20.476+0000] {override.py:1911} INFO - Created Permission View: can edit on DAG:download_rocket_launch
[2025-01-16T18:40:20.494+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:40:20.494+0000] {override.py:1911} INFO - Created Permission View: can delete on DAG:download_rocket_launch
[2025-01-16T18:40:20.505+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:40:20.505+0000] {override.py:1911} INFO - Created Permission View: can read on DAG:download_rocket_launch
[2025-01-16T18:40:20.518+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:40:20.517+0000] {override.py:1911} INFO - Created Permission View: can delete on DAG Run:download_rocket_launch
[2025-01-16T18:40:20.528+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:40:20.527+0000] {override.py:1911} INFO - Created Permission View: can read on DAG Run:download_rocket_launch
[2025-01-16T18:40:20.539+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:40:20.538+0000] {override.py:1911} INFO - Created Permission View: can create on DAG Run:download_rocket_launch
[2025-01-16T18:40:20.550+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:40:20.549+0000] {override.py:1911} INFO - Created Permission View: menu access on DAG Run:download_rocket_launch
[2025-01-16T18:40:20.550+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:40:20.550+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-01-16T18:40:20.564+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:40:20.564+0000] {dag.py:3262} INFO - Creating ORM DAG for download_rocket_launch
[2025-01-16T18:40:20.565+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:40:20.564+0000] {dag.py:4180} INFO - Setting next_dagrun for download_rocket_launch to None, run_after=None
[2025-01-16T18:40:20.589+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/download_rocket_launch.py took 0.294 seconds
[2025-01-16T18:40:23.058+0000] {processor.py:186} INFO - Started process (PID=6972) to work on /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:40:23.058+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/download_rocket_launch.py for tasks to queue
[2025-01-16T18:40:23.061+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:40:23.060+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:40:23.070+0000] {processor.py:925} INFO - DAG(s) 'download_rocket_launch' retrieved from /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:40:23.085+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:40:23.085+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-01-16T18:40:23.101+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:40:23.101+0000] {dag.py:4180} INFO - Setting next_dagrun for download_rocket_launch to None, run_after=None
[2025-01-16T18:40:23.129+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/download_rocket_launch.py took 0.077 seconds
[2025-01-16T18:40:53.193+0000] {processor.py:186} INFO - Started process (PID=7027) to work on /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:40:53.196+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/download_rocket_launch.py for tasks to queue
[2025-01-16T18:40:53.200+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:40:53.199+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:40:53.213+0000] {processor.py:925} INFO - DAG(s) 'download_rocket_launch' retrieved from /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:40:53.246+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:40:53.245+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-01-16T18:40:53.272+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:40:53.271+0000] {dag.py:4180} INFO - Setting next_dagrun for download_rocket_launch to None, run_after=None
[2025-01-16T18:40:53.306+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/download_rocket_launch.py took 0.123 seconds
[2025-01-16T18:41:23.582+0000] {processor.py:186} INFO - Started process (PID=7089) to work on /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:41:23.584+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/download_rocket_launch.py for tasks to queue
[2025-01-16T18:41:23.589+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:41:23.588+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:41:23.602+0000] {processor.py:925} INFO - DAG(s) 'download_rocket_launch' retrieved from /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:41:23.636+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:41:23.635+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-01-16T18:41:23.659+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:41:23.658+0000] {dag.py:4180} INFO - Setting next_dagrun for download_rocket_launch to None, run_after=None
[2025-01-16T18:41:23.695+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/download_rocket_launch.py took 0.122 seconds
[2025-01-16T18:41:53.836+0000] {processor.py:186} INFO - Started process (PID=7153) to work on /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:41:53.838+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/download_rocket_launch.py for tasks to queue
[2025-01-16T18:41:53.842+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:41:53.841+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:41:53.851+0000] {processor.py:925} INFO - DAG(s) 'download_rocket_launch' retrieved from /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:41:53.878+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:41:53.877+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-01-16T18:41:53.896+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:41:53.896+0000] {dag.py:4180} INFO - Setting next_dagrun for download_rocket_launch to None, run_after=None
[2025-01-16T18:41:53.924+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/download_rocket_launch.py took 0.094 seconds
[2025-01-16T18:42:24.358+0000] {processor.py:186} INFO - Started process (PID=7217) to work on /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:42:24.361+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/download_rocket_launch.py for tasks to queue
[2025-01-16T18:42:24.365+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:42:24.364+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:42:24.375+0000] {processor.py:925} INFO - DAG(s) 'download_rocket_launch' retrieved from /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:42:24.404+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:42:24.403+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-01-16T18:42:24.424+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:42:24.423+0000] {dag.py:4180} INFO - Setting next_dagrun for download_rocket_launch to None, run_after=None
[2025-01-16T18:42:24.454+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/download_rocket_launch.py took 0.104 seconds
[2025-01-16T18:42:55.384+0000] {processor.py:186} INFO - Started process (PID=7280) to work on /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:42:55.386+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/download_rocket_launch.py for tasks to queue
[2025-01-16T18:42:55.390+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:42:55.390+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:42:55.399+0000] {processor.py:925} INFO - DAG(s) 'download_rocket_launch' retrieved from /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:42:55.425+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:42:55.425+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-01-16T18:42:55.444+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:42:55.444+0000] {dag.py:4180} INFO - Setting next_dagrun for download_rocket_launch to None, run_after=None
[2025-01-16T18:42:55.475+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/download_rocket_launch.py took 0.098 seconds
[2025-01-16T18:43:25.825+0000] {processor.py:186} INFO - Started process (PID=7344) to work on /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:43:25.827+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/download_rocket_launch.py for tasks to queue
[2025-01-16T18:43:25.831+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:43:25.830+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:43:25.840+0000] {processor.py:925} INFO - DAG(s) 'download_rocket_launch' retrieved from /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:43:25.870+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:43:25.870+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-01-16T18:43:25.888+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:43:25.887+0000] {dag.py:4180} INFO - Setting next_dagrun for download_rocket_launch to None, run_after=None
[2025-01-16T18:43:25.919+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/download_rocket_launch.py took 0.102 seconds
[2025-01-16T18:43:56.013+0000] {processor.py:186} INFO - Started process (PID=7408) to work on /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:43:56.014+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/download_rocket_launch.py for tasks to queue
[2025-01-16T18:43:56.018+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:43:56.017+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:43:56.028+0000] {processor.py:925} INFO - DAG(s) 'download_rocket_launch' retrieved from /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:43:56.057+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:43:56.057+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-01-16T18:43:56.076+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:43:56.076+0000] {dag.py:4180} INFO - Setting next_dagrun for download_rocket_launch to None, run_after=None
[2025-01-16T18:43:56.108+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/download_rocket_launch.py took 0.102 seconds
[2025-01-16T18:44:26.556+0000] {processor.py:186} INFO - Started process (PID=7471) to work on /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:44:26.557+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/download_rocket_launch.py for tasks to queue
[2025-01-16T18:44:26.561+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:44:26.561+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:44:26.569+0000] {processor.py:925} INFO - DAG(s) 'download_rocket_launch' retrieved from /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:44:26.593+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:44:26.592+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-01-16T18:44:26.608+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:44:26.608+0000] {dag.py:4180} INFO - Setting next_dagrun for download_rocket_launch to None, run_after=None
[2025-01-16T18:44:26.636+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/download_rocket_launch.py took 0.087 seconds
[2025-01-16T18:44:57.349+0000] {processor.py:186} INFO - Started process (PID=7534) to work on /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:44:57.352+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/download_rocket_launch.py for tasks to queue
[2025-01-16T18:44:57.355+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:44:57.355+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:44:57.364+0000] {processor.py:925} INFO - DAG(s) 'download_rocket_launch' retrieved from /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:44:57.392+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:44:57.391+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-01-16T18:44:57.409+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:44:57.409+0000] {dag.py:4180} INFO - Setting next_dagrun for download_rocket_launch to None, run_after=None
[2025-01-16T18:44:57.444+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/download_rocket_launch.py took 0.102 seconds
[2025-01-16T18:49:12.648+0000] {processor.py:186} INFO - Started process (PID=7598) to work on /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:49:12.650+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/download_rocket_launch.py for tasks to queue
[2025-01-16T18:49:12.656+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:49:12.655+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:49:12.684+0000] {processor.py:925} INFO - DAG(s) 'download_rocket_launch' retrieved from /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:49:12.779+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:49:12.778+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-01-16T18:49:12.808+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:49:12.807+0000] {dag.py:4180} INFO - Setting next_dagrun for download_rocket_launch to None, run_after=None
[2025-01-16T18:49:12.889+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/download_rocket_launch.py took 0.267 seconds
[2025-01-16T18:49:43.085+0000] {processor.py:186} INFO - Started process (PID=7662) to work on /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:49:43.088+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/download_rocket_launch.py for tasks to queue
[2025-01-16T18:49:43.093+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:49:43.092+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:49:43.110+0000] {processor.py:925} INFO - DAG(s) 'download_rocket_launch' retrieved from /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:49:43.137+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:49:43.137+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-01-16T18:49:43.155+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:49:43.155+0000] {dag.py:4180} INFO - Setting next_dagrun for download_rocket_launch to None, run_after=None
[2025-01-16T18:49:43.374+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/download_rocket_launch.py took 0.299 seconds
[2025-01-16T18:50:14.008+0000] {processor.py:186} INFO - Started process (PID=7726) to work on /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:50:14.010+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/download_rocket_launch.py for tasks to queue
[2025-01-16T18:50:14.012+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:50:14.012+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:50:14.020+0000] {processor.py:925} INFO - DAG(s) 'download_rocket_launch' retrieved from /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:50:14.043+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:50:14.043+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-01-16T18:50:14.062+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:50:14.062+0000] {dag.py:4180} INFO - Setting next_dagrun for download_rocket_launch to None, run_after=None
[2025-01-16T18:50:14.299+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/download_rocket_launch.py took 0.297 seconds
[2025-01-16T18:50:44.640+0000] {processor.py:186} INFO - Started process (PID=7790) to work on /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:50:44.643+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/download_rocket_launch.py for tasks to queue
[2025-01-16T18:50:44.647+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:50:44.646+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:50:44.660+0000] {processor.py:925} INFO - DAG(s) 'download_rocket_launch' retrieved from /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:50:44.697+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:50:44.697+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-01-16T18:50:44.721+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:50:44.721+0000] {dag.py:4180} INFO - Setting next_dagrun for download_rocket_launch to None, run_after=None
[2025-01-16T18:50:44.762+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/download_rocket_launch.py took 0.131 seconds
[2025-01-16T18:51:15.628+0000] {processor.py:186} INFO - Started process (PID=7861) to work on /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:51:15.629+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/download_rocket_launch.py for tasks to queue
[2025-01-16T18:51:15.634+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:51:15.633+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:51:15.645+0000] {processor.py:925} INFO - DAG(s) 'download_rocket_launch' retrieved from /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:51:15.674+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:51:15.673+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-01-16T18:51:15.696+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:51:15.696+0000] {dag.py:4180} INFO - Setting next_dagrun for download_rocket_launch to None, run_after=None
[2025-01-16T18:51:15.728+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/download_rocket_launch.py took 0.109 seconds
[2025-01-16T18:51:46.356+0000] {processor.py:186} INFO - Started process (PID=7925) to work on /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:51:46.358+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/download_rocket_launch.py for tasks to queue
[2025-01-16T18:51:46.366+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:51:46.366+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:51:46.380+0000] {processor.py:925} INFO - DAG(s) 'download_rocket_launch' retrieved from /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:51:46.414+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:51:46.414+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-01-16T18:51:46.434+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:51:46.433+0000] {dag.py:4180} INFO - Setting next_dagrun for download_rocket_launch to None, run_after=None
[2025-01-16T18:51:46.466+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/download_rocket_launch.py took 0.120 seconds
[2025-01-16T18:52:17.165+0000] {processor.py:186} INFO - Started process (PID=7989) to work on /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:52:17.166+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/download_rocket_launch.py for tasks to queue
[2025-01-16T18:52:17.516+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:52:17.169+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:52:17.523+0000] {processor.py:925} INFO - DAG(s) 'download_rocket_launch' retrieved from /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:52:17.547+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:52:17.547+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-01-16T18:52:17.564+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:52:17.564+0000] {dag.py:4180} INFO - Setting next_dagrun for download_rocket_launch to None, run_after=None
[2025-01-16T18:52:17.586+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/download_rocket_launch.py took 0.429 seconds
[2025-01-16T18:52:47.758+0000] {processor.py:186} INFO - Started process (PID=8052) to work on /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:52:47.761+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/download_rocket_launch.py for tasks to queue
[2025-01-16T18:52:47.765+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:52:47.765+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:52:47.779+0000] {processor.py:925} INFO - DAG(s) 'download_rocket_launch' retrieved from /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:52:47.813+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:52:47.812+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-01-16T18:52:47.830+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:52:47.830+0000] {dag.py:4180} INFO - Setting next_dagrun for download_rocket_launch to None, run_after=None
[2025-01-16T18:52:47.862+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/download_rocket_launch.py took 0.114 seconds
[2025-01-16T18:53:18.191+0000] {processor.py:186} INFO - Started process (PID=8116) to work on /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:53:18.194+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/download_rocket_launch.py for tasks to queue
[2025-01-16T18:53:18.200+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:53:18.199+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:53:18.214+0000] {processor.py:925} INFO - DAG(s) 'download_rocket_launch' retrieved from /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:53:18.246+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:53:18.246+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-01-16T18:53:18.265+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:53:18.265+0000] {dag.py:4180} INFO - Setting next_dagrun for download_rocket_launch to None, run_after=None
[2025-01-16T18:53:18.299+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/download_rocket_launch.py took 0.117 seconds
[2025-01-16T18:53:48.531+0000] {processor.py:186} INFO - Started process (PID=8180) to work on /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:53:48.533+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/download_rocket_launch.py for tasks to queue
[2025-01-16T18:53:48.536+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:53:48.535+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:53:48.544+0000] {processor.py:925} INFO - DAG(s) 'download_rocket_launch' retrieved from /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:53:48.564+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:53:48.563+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-01-16T18:53:48.578+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:53:48.578+0000] {dag.py:4180} INFO - Setting next_dagrun for download_rocket_launch to None, run_after=None
[2025-01-16T18:53:48.600+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/download_rocket_launch.py took 0.074 seconds
[2025-01-16T18:54:19.401+0000] {processor.py:186} INFO - Started process (PID=8244) to work on /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:54:19.403+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/download_rocket_launch.py for tasks to queue
[2025-01-16T18:54:19.406+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:54:19.405+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:54:19.413+0000] {processor.py:925} INFO - DAG(s) 'download_rocket_launch' retrieved from /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:54:19.437+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:54:19.436+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-01-16T18:54:19.454+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:54:19.454+0000] {dag.py:4180} INFO - Setting next_dagrun for download_rocket_launch to None, run_after=None
[2025-01-16T18:54:19.477+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/download_rocket_launch.py took 0.082 seconds
[2025-01-16T18:54:49.704+0000] {processor.py:186} INFO - Started process (PID=8308) to work on /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:54:49.706+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/download_rocket_launch.py for tasks to queue
[2025-01-16T18:54:49.709+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:54:49.709+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:54:49.722+0000] {processor.py:925} INFO - DAG(s) 'download_rocket_launch' retrieved from /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:54:49.750+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:54:49.750+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-01-16T18:54:49.774+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:54:49.774+0000] {dag.py:4180} INFO - Setting next_dagrun for download_rocket_launch to None, run_after=None
[2025-01-16T18:54:49.806+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/download_rocket_launch.py took 0.109 seconds
[2025-01-16T18:55:20.595+0000] {processor.py:186} INFO - Started process (PID=8372) to work on /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:55:20.597+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/download_rocket_launch.py for tasks to queue
[2025-01-16T18:55:20.602+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:55:20.601+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:55:20.614+0000] {processor.py:925} INFO - DAG(s) 'download_rocket_launch' retrieved from /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:55:20.641+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:55:20.641+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-01-16T18:55:20.662+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:55:20.661+0000] {dag.py:4180} INFO - Setting next_dagrun for download_rocket_launch to None, run_after=None
[2025-01-16T18:55:20.693+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/download_rocket_launch.py took 0.106 seconds
[2025-01-16T18:55:51.607+0000] {processor.py:186} INFO - Started process (PID=8436) to work on /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:55:51.609+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/download_rocket_launch.py for tasks to queue
[2025-01-16T18:55:51.614+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:55:51.612+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:55:51.625+0000] {processor.py:925} INFO - DAG(s) 'download_rocket_launch' retrieved from /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:55:51.656+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:55:51.656+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-01-16T18:55:51.675+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:55:51.675+0000] {dag.py:4180} INFO - Setting next_dagrun for download_rocket_launch to None, run_after=None
[2025-01-16T18:55:51.709+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/download_rocket_launch.py took 0.110 seconds
[2025-01-16T18:57:28.665+0000] {processor.py:186} INFO - Started process (PID=37) to work on /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:57:28.676+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/download_rocket_launch.py for tasks to queue
[2025-01-16T18:57:28.685+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:57:28.684+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:57:28.902+0000] {processor.py:925} INFO - DAG(s) 'download_rocket_launch' retrieved from /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:57:29.145+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:57:29.145+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-01-16T18:57:29.215+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:57:29.214+0000] {dag.py:4180} INFO - Setting next_dagrun for download_rocket_launch to None, run_after=None
[2025-01-16T18:57:29.311+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/download_rocket_launch.py took 0.711 seconds
[2025-01-16T18:57:59.755+0000] {processor.py:186} INFO - Started process (PID=101) to work on /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:57:59.757+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/download_rocket_launch.py for tasks to queue
[2025-01-16T18:57:59.761+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:57:59.760+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:57:59.774+0000] {processor.py:925} INFO - DAG(s) 'download_rocket_launch' retrieved from /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:57:59.809+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:57:59.808+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-01-16T18:57:59.832+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:57:59.831+0000] {dag.py:4180} INFO - Setting next_dagrun for download_rocket_launch to None, run_after=None
[2025-01-16T18:57:59.868+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/download_rocket_launch.py took 0.123 seconds
[2025-01-16T18:58:30.064+0000] {processor.py:186} INFO - Started process (PID=165) to work on /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:58:30.066+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/download_rocket_launch.py for tasks to queue
[2025-01-16T18:58:30.071+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:58:30.070+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:58:30.083+0000] {processor.py:925} INFO - DAG(s) 'download_rocket_launch' retrieved from /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:58:30.143+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:58:30.143+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-01-16T18:58:30.183+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:58:30.182+0000] {dag.py:4180} INFO - Setting next_dagrun for download_rocket_launch to None, run_after=None
[2025-01-16T18:58:30.233+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/download_rocket_launch.py took 0.192 seconds
[2025-01-16T18:59:01.062+0000] {processor.py:186} INFO - Started process (PID=229) to work on /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:59:01.065+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/download_rocket_launch.py for tasks to queue
[2025-01-16T18:59:01.068+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:59:01.068+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:59:01.078+0000] {processor.py:925} INFO - DAG(s) 'download_rocket_launch' retrieved from /opt/airflow/dags/download_rocket_launch.py
[2025-01-16T18:59:01.106+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:59:01.106+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-01-16T18:59:01.124+0000] {logging_mixin.py:190} INFO - [2025-01-16T18:59:01.123+0000] {dag.py:4180} INFO - Setting next_dagrun for download_rocket_launch to None, run_after=None
[2025-01-16T18:59:01.157+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/download_rocket_launch.py took 0.101 seconds
